recipes/_environment.rb

```diff
@@ -107,6 +108,24 @@
     owner node['omnibus']['build_user']
     group node['omnibus']['build_user_group']
   end
+
+  powershell_file = batch_file.gsub('.bat', '.ps1')
+  file powershell_file do
+    content <<-EOH.gsub(/^ {6}/, '')
+      $tempFile = [IO.Path]::GetTempFileName()
+      cmd /c " `"#{batch_file}`" && set > `"$tempFile`" "
+      Get-Content $tempFile | Foreach-Object {
+        if($_ -match "^(.*?)=(.*)$")
+        {
+          Set-Content "env:\\$($matches[1])" $matches[2]
+        }
+      }
+
+      Remove-Item $tempFile
+    EOH
```

Comment:
I'd prefer we skip any `*.bat` file loading magic and just set the environment variables directly in this PowerShell script like we do here:
https://github.com/chef-cookbooks/omnibus/blob/5a3b21582bfee5649f0026f0c2b971ef97d8e321/recipes/_environment.rb#L32-L35